jujugui/static/gui/src/app/utils/changes-utils.js

```diff
@@ -0,0 +1,242 @@
+            break;
+          case '_addMachines':
+            var machineType = change.command.args[0][0].parentId ?
+                'container' : 'machine';
+            changeItem.icon = 'changes-' + machineType + '-created';
+            changeItem.description = change.command.args[0].length +
+                ' ' + machineType +
+                (change.command.args[0].length !== 1 ? 's have' : ' has') +
+                ' been added.';
+            break;
+          case '_destroyMachines':
+            /*jshint -W004*/
+            var machineType = change.command.args[0][0].indexOf('/') !== -1 ?
+                'container' : 'machine';
+            changeItem.icon = 'changes-' + machineType + '-destroyed';
+            changeItem.description = change.command.args[0].length +
+                ' ' + machineType +
+                (change.command.args[0].length !== 1 ? 's have' : ' has') +
+                ' been destroyed.';
+            break;
+          case '_set_config':
+            var cfgServ = services.getById(change.command.args[0]);
+            changeItem.icon = 'changes-config-changed';
+            changeItem.description = 'Configuration values changed for ' +
+                cfgServ.get('displayName').match(removeBrackets)[1] + '.';
+            break;
+          default:
+            changeItem.icon = 'changes-service-exposed';
+            changeItem.description = 'An unknown change has been made ' +
+                'to this enviroment via the CLI.';
+            break;
+        }
+      }
+      if (skipTime || !change) {
+        changeItem.time = '00:00';
+      } else {
+        changeItem.time = this._formatAMPM(new Date(change.timestamp));
+      }
+      return changeItem;
+    },
+
+    /**
+      Return formatted time for display.
+
+      @method _formatAMPM
+      @param {Date} date The current date.
+    */
+    _formatAMPM: function(date) {
+      var hours = date.getHours();
+      var minutes = date.getMinutes();
+      var ampm = hours >= 12 ? 'pm' : 'am';
+      hours = hours % 12;
+      hours = hours ? hours : 12;
+      minutes = minutes < 10 ? '0' + minutes : minutes;
+      var strTime = hours + ':' + minutes + ' ' + ampm;
+      return strTime;
+    },
+
+    /**
+      Loops through the services in the db to find ones which have id's which
+      match the temporary id's assigned to the add realation call.
+
+      @method getRealRelationEndpointNames
+      @param {Array} args The arguments array from the ecs add relations call.
+      @return {Array} An array of the service names involved in the relation.
+    */
+    getRealRelationEndpointNames: function(args, services) {
+      var serviceList = [],
+          serviceId;
+      services.each(function(service) {
+        serviceId = service.get('id');
+        args.forEach(function(arg) {
+          if (serviceId === arg[0]) {
+            serviceList.push(
+                service.get('displayName').replace(/^\(|\)$/g, ''));
```

Comment:
I think you meant to use the `removeBrackets` regex here which isn't being used anywhere that I can see